### PR TITLE
Updates to Reflected Cross-Site Scripting

### PR DESCRIPTION
**Pull request by RRudder**
High level finding updated based on peer review
* updates to overview, steps to repro, business impact
* removal of placeholder text

### CODE REVIEW

File: submissions/description/cross_site_scripting_xss/reflected/recommendations.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Recommendation(s)
+
+There is no single technique to stop XSS from occurring. However, implementing the right combination of defensive measures within the application will prevent and limit the impact of XSS. Some best practices include the following:
+
+- All user input fields should be sanitized based on what the field is likely to contain. For example, a date field (01/01/2001) should only contain a maximum of 10 characters consisting of numbers and forward slashes. Additionally, drop down or pick lists can be used for allowable inputs to ensure expected values are sent to the server.
+- Use appropriate HTTP response headers to ensure the browser correctly interprets responses. These should be customized specific to the application and its environment. For example:
+
+``` bash
```
Comment (vortexau) [inline submissions/description/cross_site_scripting_xss/reflected/recommendations.md:8]: Strictly `bash` isn't correct, this should be `HTTP` as these are headers.  Can be fixed in the future, though.

File: submissions/description/cross_site_scripting_xss/reflected/template.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Reflected XSS could lead to data theft through the attacker’s ability to manipulate data through their access to the application, and their ability to interact with other users, including performing other malicious attacks, which would appear to originate from a legitimate user. These malicious actions could also result in reputational damage for the business through the impact to customers’ trust.
+
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Use a browser to navigate to: {{URL}}
+1. Forward the following request to the endpoint:
+
+```HTTP Request
```
Comment (vortexau) [inline submissions/description/cross_site_scripting_xss/reflected/template.md:19]: This one should also be `HTTP` as in the past, we'll need to fix this one as this is incorrect syntax.

**Review by vortexau — CHANGES_REQUESTED** [review]
A couple of changes with syntax highlighting to resolve.

File: submissions/description/cross_site_scripting_xss/reflected/recommendations.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/reflected/recommendations.md]: Thanks @vortexau, I made the change here and will update in the other findings where this error is

File: submissions/description/cross_site_scripting_xss/reflected/template.md (hunk 2 above)
Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/reflected/template.md]: Updated :)

**Review by vortexau — APPROVED** [review]

**Pull request merged**